pkg/cmd/cli/cmd/buildlogs.go

```diff
@@ -17,11 +17,14 @@ import (
 )
 
 const (
-	buildLogsLong = `Retrieve logs from the containers where the build occurred.
+	buildLogsLong = `
+Retrieve logs for a build
 
-NOTE: This command may be moved in the future.`
```

Comment:
Unrelated to this, but are we going to make an "oc build <subcommand>" branch to eventually retire these top level pieces (start and cancel come to mine)?